webpack/assets/javascripts/bundle.js

```diff
@@ -56,4 +57,5 @@ window.tfm = Object.assign(window.tfm || {}, {
   componentRegistry,
   store,
   lookupKeys,
+  Cookies,
```

Comment:
Is this here so that non-react pages can access and use it?

Reply:
yes